syncd/CommandLineOptionsParser.cpp

```diff
@@ -42,6 +42,7 @@ std::shared_ptr<CommandLineOptions> CommandLineOptionsParser::parseCommandLine(
             { "contextContig",           required_argument, 0, 'x' },
             { "breakConfig",             required_argument, 0, 'b' },
             { "watchdogWarnTimeSpan",    optional_argument, 0, 'w' },
+            { "SupportingBulkCounters",  required_argument, 0, 'B' },
```

Comment:
this actually should be lower case, look at sorrounded variables

Reply:
Will fix it.

Reply:
Fixed